### PR TITLE
docs: replace gulp favicon with svg gulp logo

### PR DESCRIPTION
**Pull request by mikeattara**
Currently, the logo is the favicon for the gulp website. I have therefore replaced that with an svg version of the logo.

### CODE REVIEW

File: v1/website/data/users.js [hunk 1]
```diff
@@ -274,6 +281,13 @@ module.exports = [
     fbOpenSource: false,
     pinned: false,
   },
+  {
+    caption: 'React Native Sensors',
+    image: '/image/users/react-native-sensors.svg',
```
Comment (yangshun) [inline v1/website/data/users.js]: This should be `/img/`

**Comment by yangshun** [comment]
@mikeattara This PR conflicts with the other one you made. Could you resolve the conflict and update this PR?

File: v1/website/data/users.js (hunk 1 above)
Comment (mikeattara) [reply v1/website/data/users.js]: Awwwsh, I usually name my folders with the full name.

**Comment by mikeattara** [comment]
@yangshun I have resolved the merge conflict

**Comment by yangshun** [comment]
Thanks @mikeattara!

**Pull request merged**